src/main/java/DukeCommandValidator.java

```diff
@@ -0,0 +1,126 @@
+public class DukeCommandValidator {
+    static int MINIMUM_DONE_LENGTH = 5;
+    static int MINIMUN_TODO_LENGTH = 6;
+    static int MINIMUM_EVENT_LENGTH = 12;
+    static int MINIMUM_DEADLINE_LENGTH = 15;
+
+    public static int getCommand(String input){
+        if(input.equals("bye")){
+            return DukeCommands.EXIT;
+        }
+        if(input.equals("list")){
+            return DukeCommands.LIST;
+        }
+        if(validateDoneCommand(input) == true){
+            return DukeCommands.DONE;
+        }
+        if(validateAddCommand(input) == true) {
+            return DukeCommands.ADD;
+        }
+        return DukeCommands.INVALID_COMMAND;
+    }
+
+    public static boolean validateTodo(String input){
```

Comment:
May want to change to something like "isTodoValidated" to make it sound more like a boolean